content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Monitoring Linux host metrics with the Node Exporter
+---
+
+# Monitoring Linux host metrics with the Node Exporter
+
+The Prometheus [**Node Exporter**](https://github.com/prometheus/node_exporter) exposes a wide variety of hardware- and kernel-related metrics.
+
+In this guide, you will:
+
+* Start up a Node Exporter on `localhost`
+* Start up a Prometheus instance on `localhost` that's configured to scrape metrics from the running Node Exporter
+
+NOTE: While the Prometheus Node Exporter is for *nix systems, there is a [WMI exporter](https://github.com/martinlindhe/wmi_exporter) for Windows that serves an analogous purpose.
+
+## Installing and running the Node Exporter
+
+The Prometheus Node Exporter is a single static binary that you can install [via tarball](#tarball-installation). You can [download](/downloads#node_exporter) page, extract it, and run it:
```

Comment:
You can download page? 

Partial sentence?

Reply:
Sorry, this doc is in the middle of a re-write, hence the odd loose ends. I'll get these straightened up.